migrations/db/migrations/20230607094822_add_auth_schema_protections.sql

```diff
@@ -98,18 +111,17 @@ create or replace function
     end;
   $$;
 
-drop event trigger if exists protect_auth_schema_ddl_command_end cascade;
-
 create event trigger
   protect_auth_schema_ddl_command_end
   on ddl_command_end
   execute function auth.schema_ddl_protection_command_end();
 
-drop event trigger if exists protect_auth_schema_ddl_sql_drop cascade;
-
 create event trigger
   protect_auth_schema_ddl_sql_drop
   on sql_drop
   execute function auth.schema_ddl_protection_drop();
 
+-- Enable the lock
+select auth.enable_schema_ddl_protection()
```

Comment:
note, security enabled after migration runs

Reply:
Let's keep it off for now, we can roll it out individually on projects as a first stage, then once confident it's a good idea we can enable it in the migrations.